org.pitivi.Pitivi.json

```diff
@@ -0,0 +1,462 @@
+                }
+            ]
+        },
+        {
+            "name": "x265",
+            "buildsystem": "cmake",
+            "subdir":"source",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://bitbucket.org/multicoreware/x265/downloads/x265_1.9.tar.gz",
+                    "sha256": "3e4654133ed957a98708fdb4cb9a154d9e80922b84e26e43fc462a101c5b15c8"
+                }
+            ]
+        },
+        {
+            "name": "ffmpeg",
+            "config-opts": [
+                "--disable-static",
+                "--disable-avdevice",
+                "--disable-postproc",
+                "--disable-programs",
+                "--disable-ffserver",
+                "--disable-ffplay",
+                "--disable-ffprobe",
+                "--disable-ffmpeg",
+                "--disable-encoder=flac",
+                "--disable-protocols",
+                "--disable-devices",
+                "--disable-network",
+                "--disable-hwaccels",
+                "--disable-dxva2",
+                "--disable-vdpau",
+                "--disable-filters",
+                "--disable-doc",
+                "--disable-vda",
+                "--disable-d3d11va",
+                "--disable-dxva2",
+                "--disable-audiotoolbox",
+                "--disable-videotoolbox",
+                "--disable-vaapi",
+                "--disable-crystalhd",
+                "--disable-mediacodec",
+                "--disable-nvenc",
+                "--disable-mmal",
+                "--disable-omx",
+                "--disable-omx-rpi",
+                "--disable-cuda",
+                "--disable-cuvid",
+                "--disable-libmfx",
+                "--disable-libnpp",
+                "--disable-iconv",
+                "--disable-jni",
+                "--enable-optimizations",
+                "--enable-gpl",
+                "--enable-shared"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://git.ffmpeg.org/ffmpeg.git",
+                    "branch": "e3a1c0491fa2beb6562f9960e9cb3af673f3bf94"
+                }
+            ]
+        },
+        {
+            "name": "fdkaac",
+            "config-opts": ["--disable-static"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/mstorsjo/fdk-aac.git",
+                    "branch": "v0.1.5",
+                    "commit": "74c1a2a4f831285cbd93ec1427f1670d3c5c5e52"
+                }
+            ]
+        },
+        {
+            "name": "gstreamer",
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts": ["-Ddisable_gtkdoc=true", "--libdir=lib"],
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "1.12.3",
```

Comment:
If you want 1.12 now the 3.26 runtime is out, so why wouldn't you use that.

Reply:
"We need our own build of Gst as we have backported patches that are not in the 1.12 branch and we build quite many more plugins than what is provided in the Sdk."